app/Http/Controllers/Service/RegistrationController.php

```diff
@@ -151,38 +152,104 @@ public function edit($id)
      * @param integer $id
      * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
      */
-    public function print($id)
+    public function printOneIndividualFamilyForm($id)
     {
+        // Get User
         $user = Auth::user();
 
+        // Find the Registration and subdata
         $registration = Registration::with([
             'family' => function ($q) {
                 $q->with('children', 'carers');
             }
         ])->findOrFail($id);
 
-        if (!$registration) {
-            // Todo better error handling - but for now, just return is better than 500.
-            // We don't expect this to happen- ever... but
-            return;
-        }
-
-        $carers = $registration->family->carers->all();
+        // Make a filename
+        $filename = 'Registration' . Carbon::now()->format('YmdHis') .'.pdf';
 
-        $params = [
+        // Setup common data
+        $data = [
             'user_name' => $user->name,
             'centre_name' => ($user->centre) ? $user->centre->name : null,
-            'centre' => $registration->centre,
             'sheet_title' => 'Printable Family Sheet',
             'sheet_header' => 'Family Collection Sheet',
+        ];
+
+        $carers = $registration->family->carers->all();
```

Comment:
This will throw a 500 if the family isn't found in the findOrFail on ln 165. I think we either stick with find (if we 'know' it will always be there) or put a check in after we look for it to abort if not found.

Reply:
agreed.